### ISSUE TITLE
Add println(Object...) to io.vavr.API

### ISSUE THREAD

**Issue by danieldietrich** [vavr-io/vavr#2381]
VAVR already has println(Object):

```java
import static io.vavr.API.*;
...
// prints test
println("test")
```

**Goal:** Add add println(Object...)

```java
import static io.vavr.API.*;
...
// prints this and that
println("this", "and", "that")
```

Please note, that

* the objects are stringified
* the objects are separated by space ' '

_The attentive reader may have noticed that there is some resemblance to JavaScript's console.log ;)_

**Comment by thadumi** [comment]
Into the note section, you said that the objects are stringified. Are we assuming that the method toString is defined or should we implement a stringify function which works similarly to the JS one?
I already saw that the actual version of println just calls the System.out.println method and therefore is using the toString solution.

**Comment by danieldietrich** [comment]
Yes, we use toString() - or more specifically: String.valueOf(), which handles the null case.

A solution could look like this:

```java
// io.vavr.collection.Iterator
Iterator.of(xs).map(String::valueOf).mkString(" ")
```

**Comment by agebhar1** [comment]
This issue might be closed due to #2386.

**Comment by danieldietrich** [comment]
true, thank you!
